library/list.lisp

```diff
@@ -597,9 +615,9 @@ This function is equivalent to all size-N elements of `(COMBS L)`."
           (True (match l
                   ((Nil) Nil)
                   ((Cons x xs) (append
-                              (map (Cons x) (combsOf (- n 1) xs)) ; combs with X
-                              (combsOf n xs)                      ; and without x
-                              ))))))
+                                (map (Cons x) (combsOf (- n 1) xs)) ; combs with X
+                                (combsOf n xs)))))))                      ; and without x
```

Comment:
i have no strong feelings about putting the parens on the line or after it, but could you keep the comments lined up?